directory/datadog_checks/directory/traverse.py

```diff
@@ -0,0 +1,60 @@
+import platform
+import sys
+from os.path import join
+
+import six
+from scandir import scandir
+
+
+def _walk(top):
+    """Modified version of https://docs.python.org/3/library/os.html#os.scandir
```

Comment:
Assuming this function is extensively tested in its original version, would it make sense to add tests for the modifications we made?

Reply:
I think not, actually, because our current tests are already doing a good job of this.